### PR TITLE
Apm 12040 upgrade bgpalerter to 1 28 4

### PR DESCRIPTION
**Pull request by jeff-leong**
Changes:
- merge 1.28.4 release branch into our fork
- update Docker build script to use node 14.x
[APM-12040](https://appneta.atlassian.net/browse/APM-12040)

### CODE REVIEW

**Comment by jsherin** [comment]
@jeff-leong is there anything that can be reviewed in this?

File: setup_build_environment.sh [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
-apt-get update
+#apt-get update
```
Comment (sheniatung) [inline setup_build_environment.sh:2]: NIT: can remove this line

**Review by sheniatung — APPROVED** [review]
New commit looks good!

**Review by sheniatung — APPROVED** [review]

**Comment by jeff-leong** [comment]
> @jeff-leong is there anything that can be reviewed in this?

These are the two commits to review
https://github.com/appneta/BGPalerter/pull/5/commits/979338bb1bf67e75c96c703d5e41047fb9c3e866
https://github.com/appneta/BGPalerter/pull/5/commits/99ceefe7a0057be7b080e01ef951a0c752fd9933

**Review by jsherin — APPROVED** [review]
:+1:

**Review by gilbertman — APPROVED** [review]
looks good!

**Pull request merged**